README.md

```diff
@@ -76,6 +76,7 @@ Yaml Options:
 | show-button-users         | object[] | **optional**  | *                      | Choose the users that button is visible to them       |
 | start-expanded-users      | object[] | **optional**  | *                      | Choose the users that card will be start expanded for them|
 | animation                 | boolean   | _true_       | true\|false            | Should the opening/closing of expander be animated? |
+| expander-card-id          | string    | **optional** | *                      | An id to use with [Set state via action](#set-state-via-action)        |
```

Comment:
We have already a storage-id. Do you think we can reuse the storage-id for both cases, then we would have less configuration

Reply:
Maybe migrate to a new config variable. storage-id is a strange config name for this function. expander-card-id could be used for all features where an id is used. 

Also, if we combine ids, realistically there should then be an option as whether to store state. So you still end up with an extra config, and then also migration for those using storage-id.